swift_browser_ui_frontend/wasm/src/include/upcommon.h

```diff
@@ -0,0 +1,52 @@
+/*
+Upload process common functions
+*/
+
+
+#include <stdint.h>
+#include "uptypes.h"
+
+
+#ifndef SWIFT_UI_UPLOAD_COMMON
+#define SWIFT_UI_UPLOAD_COMMON
+
+
+/*
+Key init function, from libcrypt4gh
+*/
+uint8_t* crypt4gh_session_key_new(void);
```

Comment:
i think there are some syntax errors in these C files e.g.
```suggestion
uint8_t *crypt4gh_session_key_new(void);
```

Reply:
Not really a syntax issue, but a styling one – but I went through the C code and fixed all instances of these 👍🏼